pkg/defaultcomponents/defaults_test.go

```diff
@@ -104,25 +104,26 @@ func TestComponents(t *testing.T) {
 	assert.NotNil(t, processors["tail_sampling"])
 	assert.NotNil(t, processors["k8sattributes"])
 
-	// Ensure that the components behind feature gates aren't included
-	assert.Nil(t, receivers["filelog"])
-	assert.Nil(t, exporters["awscloudwatchlogs"])
-	assert.Nil(t, extensions["file_storage"])
+	// Ensure that the components behind feature gates are included in stageBeta by default
+	assert.NotNil(t, receivers["filelog"])
+	assert.NotNil(t, exporters["awscloudwatchlogs"])
+	assert.NotNil(t, extensions["file_storage"])
 }
 
-func TestEnabledFeatureGate(t *testing.T) {
+// TODO : Modify this test to check the `error` is received when featuregate is in StageStable state.
```

Comment:
The Gate will be disabled by default and will return an error if modified, in stageBeta. Added this note just for track/visibility for next phase.